docs/guides/serverless-run-first-workload.ipynb

```diff
@@ -0,0 +1,396 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6ec2a7ce-6d72-401c-aecd-59f2f519153a",
+   "metadata": {},
+   "source": [
+    "# Run your first Serverless workload remotely\n",
+    "\n",
+    "This section explores how to list programs available in Serverless, pass inputs into these programs, run them remotely, check their status, and retrieve results and logs.\n",
```

Comment:
I'm not sure how I feel about just referring to Qiskit Serverless as "Serverless", I think it would be better to use the full name

Reply:
Makes sense!